### PR TITLE
Features/pf 2602 move to tiny mce

### PR DESCRIPTION
**Pull request by Adrickle**

### CODE REVIEW

File: Source/ProjectFirma.Web/Common/TinyMCEEditorExtensionTest.GenerateJavascriptWithAllOnOneRowNoMaximizeToolbarsTest.received.txt
Comment (tomkamin) [inline Source/ProjectFirma.Web/Common/TinyMCEEditorExtensionTest.GenerateJavascriptWithAllOnOneRowNoMaximizeToolbarsTest.received.txt:1]: you shouldn't commit the received file. running the unit test should generate this and match the approved version

File: Source/ProjectFirma.Web/Controllers/FileResourceController.cs [hunk 1]
```diff
@@ -150,175 +150,5 @@ public ActionResult GetFileResourceResized(string fileResourceGuidAsString, int
...
-        public ContentResult CkEditorUploadFileResource(FirmaPagePrimaryKey firmaPagePrimaryKey)
-        {
-            return Content(String.Empty);
-        }
-
-        [CrossAreaRoute]
-        [HttpPost]
-        [AutomaticallyCallEntityFrameworkSaveChangesWhenModelValid]
-        [FirmaPageManageFeature]
-        public ContentResult CkEditorUploadFileResource(FirmaPagePrimaryKey firmaPagePrimaryKey, CkEditorImageUploadViewModel viewModel)
```
Comment (tomkamin) [inline Source/ProjectFirma.Web/Controllers/FileResourceController.cs:170]: With this function removed, along with the others from this file, how are file uploads handled?

**Review by tomkamin — CHANGES_REQUESTED** [review]
I cant get any rich text editors to load, using the NCRP tenant to test.
This error is in the console:
Uncaught SyntaxError: missing } after property list

File: Source/ProjectFirma.Web/Common/TinyMCEEditorExtensionTest.GenerateJavascriptWithAllOnOneRowNoMaximizeToolbarsTest.received.txt
Comment (Adrickle) [reply Source/ProjectFirma.Web/Common/TinyMCEEditorExtensionTest.GenerateJavascriptWithAllOnOneRowNoMaximizeToolbarsTest.received.txt:1]: fixed!

**Comment by Adrickle** [comment]
> I cant get any rich text editors to load, using the NCRP tenant to test. This error is in the console: Uncaught SyntaxError: missing } after property list

This should be fixed now

File: Source/ProjectFirma.Web/Controllers/FileResourceController.cs (hunk 1 above)
Comment (tomkamin) [reply Source/ProjectFirma.Web/Controllers/FileResourceController.cs:170]: We should document that and consider moving all images currently in rich text fields into the text field like tiny MCE does, then we can remove the DB tables storing the files.

**Review by tomkamin — CHANGES_REQUESTED** [review]
If you can, please use the tinyMCE nuget package. Otherwise looks good and can merge!

File: Source/ProjectFirma.Web/wwwroot/lib/tinymce/tinymce.js
Comment (tomkamin) [inline Source/ProjectFirma.Web/wwwroot/lib/tinymce/tinymce.js:1]: I am not sure how tinymce got into a wwwroot folder, but it looks like it is installed twice.
I tested on my local, uninstalled TinyMCE from nuget which left the wwwroot folder around. So I manually deleted that and reinstalled TinyMCE using the nuget GUI.
I have a feeling it was due to your initial install using the command line. Lets remove the whole wwwroot folder. It is not needed.

**Review by tomkamin — CHANGES_REQUESTED** [review]

**Review by tomkamin — APPROVED** [review]

**Pull request merged**